distributions/stacks/openshift/application/openshift-service-mesh/controlplane.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  namespace: istio-system
+  name: basic
+spec:
```

Comment:
Can we give it a meaningful name instead of "basic" something like opendathub or kubeflow control plane?

Reply:
Updated to `kubeflow`